hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/receiver/kafka/MessageContentReader.java

```diff
@@ -0,0 +1,9 @@
+package pl.allegro.tech.hermes.consumers.consumer.receiver.kafka;
+
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import pl.allegro.tech.hermes.api.ContentType;
+import pl.allegro.tech.hermes.common.message.wrapper.UnwrappedMessageContent;
+
+public interface MessageContentReader {
```

Comment:
Shouldnt' this be `MessageContentUnwrapper` as it the main point for returning `UnwrappedMessageContent`? :)

Reply:
I think from the perspective of the internal implementation the most appropriate would be
```
public interface MessageContentUnwrapper {
    UnwrappedMessageContent unwrap(ConsumerRecord<byte[], byte[]> message, ContentType contentType);
}
```
but since the interface is dedicated for external users who want to provide custom implementations of reading messages without "unwrapping", we decided to choose this name.

Reply:
OK, I agree. :)